### PR TITLE
Add telcov10n config for project2.

### PR DESCRIPTION
**Pull request by semihkisa**
PoC for Solution Lifecycle Management tests in Openshift CI for project2. 

Details under the Jira task: TELCOV10N-368

### CODE REVIEW

**Comment by natifridman** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-slcm-gitlab-project2-site3-eco-ci-cd

**Review by cplacani — APPROVED** [review]

**Review by natifridman — APPROVED** [review]
/lgtm

File: ci-operator/config/openshift-kni/eco-ci-cd/OWNERS [hunk 1]
```diff
@@ -9,9 +9,11 @@ approvers:
 - cplacani
 - natifridman
 - shaior
+- semihkisa
```
[the remark this answers is not in the document]
Comment (semihkisa) [reply ci-operator/config/openshift-kni/eco-ci-cd/OWNERS]: Indeed the file suggests not to edit, so I removed my name. Thanks .

**Pull request merged**